src/DataCollection.WPF/MainWindow.xaml

```diff
@@ -222,26 +244,49 @@
                 <!--  Open Menu Button  -->
                 <ToggleButton
                     x:Name="toggleMenu"
-                    Width="60"
+                    Width="60" Click="menuButton_Click"
                     HorizontalAlignment="Right"
                     Background="{Binding ConnectivityMode, Converter={StaticResource ConnectivityModeToColorConverter}}"
                     Content="&#xE712;"
                     FontFamily="Segoe MDL2 Assets"
-                    FontSize="25"
+                    FontSize="20"
                     Style="{StaticResource ToggleButtonStyle}"
                     ToolTip="{Binding ., Converter={StaticResource LocalizationConverter}, ConverterParameter=MoreTools_Tooltip}"
                     Visibility="{Binding IsLocationOnlyMode, Converter={StaticResource BoolToVisibilityConverter}, ConverterParameter=Inverse}" />
             </StackPanel>
         </Grid>
 
         <!--  Menu Panel  -->
+        <Grid Grid.Row="1"
```

Comment:
I feel like the TOC would really benefit from some type of border. It feels unfinished with the white background with no rounded corners.